### PR TITLE
Allow setting LineRange to null in NotePosition

### PR DESCRIPTION
**Pull request by KqLLL**
Fix [Issue 1536](https://github.com/xanzy/go-gitlab/issues/1536)

### CODE REVIEW

File: notes.go [hunk 1]
```diff
@@ -73,6 +73,8 @@ type Note struct {
 }
 
 // NotePosition represents the position attributes of a note.
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/discussions.html#create-new-merge-request-thread
```
Comment (svanharmelen) [inline notes.go:77]: Lets omit these lines as this struct is not used only for the API call.

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Please remove the additional comments, the code change itself is fine 👍🏻

File: notes.go (hunk 1 above)
Comment (KqLLL) [reply notes.go]: 👌

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

**Pull request merged**